adal/src/main/java/com/microsoft/aad/adal/IBrokerProxy.java

```diff
@@ -39,7 +39,9 @@
      */
     BrokerProxy.SwitchToBroker canSwitchToBroker(final String authorityUrlStr);
 
-    boolean verifyUser(String username, String uniqueid);
+    boolean verifyUser(String username, String uniqueid) throws AuthenticationException;
```

Comment:
Is username the upn? UserId? Email?

Reply:
This is always used with loginHint currently.  Login hint is expected to be upn in almost every case.  I didn't actually add this method.... so i'm going to leave as is for now.